docs/src/account/index.md

```diff
@@ -10,37 +14,39 @@ In Miden's hybrid UTXO- and account-based model, accounts enable the creation of
 
 An `Account` is composed of several core parts, illustrated below:
 
-<p style="text-align: center;">
-    <img src="../img/account/account-definition.png" style="width:30%;" alt="Account diagram"/>
+<p style={{textAlign: 'center'}}>
+    <img src={require('../img/account/account-definition.png').default} style={{width: '30%'}} alt="Account diagram"/>
 </p>
 
 These parts are:
 
-1. [ID](id.md)
-2. [Code](code.md)
-3. [Storage](storage.md)
+1. [ID](id)
+2. [Code](code)
+3. [Storage](storage)
 4. [Vault](#vault)
 5. [Nonce](#nonce)
 
 ### Vault
 
-> [!Note]
-> A collection of [assets](../asset.md) stored by the `Account`.
+:::note
```

Comment:
This `:::note` looks weird here and in other places. I think our initial intention was to give a definition for the datastructure, e.g., `Vault`, and the an explanation. But since there are only two sentences, let's try to get rid of the `:::note` here and in other places.

<img width="1320" height="380" alt="Screenshot 2025-09-30 at 11 05 16" src="https://github.com/user-attachments/assets/c43b35fc-4f3b-42fe-bb09-663059c31c4d" />